theme/theme.inc

```diff
@@ -401,37 +378,15 @@ function template_process_islandora_newspaper_page(array &$variables) {
       $variables['content'] = NULL;
     }
   }
+  // Fallback to image.
```

Comment:
I pulled this down and it works as expected.  The two code blocks under both // Fallback to image comments seem identical so there maybe an opportunity to refactor, but I'm willing to merge if others are ok with it.  Should we wait for @mjordan to comment.